README.md

```diff
@@ -31,12 +31,112 @@ SRT is applied to contribution and distribution endpoints as part of a video str
+
+  </p>
+</details>
+
+<details>
+  <summary>Content Agnostic</summary>
+
+  <p>
+
+  Unlike some other streaming protocols that only support specific video and audio formats, SRT is payload agnostic. Because SRT operates at the network transport level, acting as a wrapper around your content, it can transport any type of video format, codec, resolution, or frame rate.
+
+  </p>
+</details>
+
+<details>
+  <summary>Easy Firewall Traversal with Rendezvous Mode</summary>
+
+  <p>
+
+  The handshaking process used by SRT supports outbound connections without the potential risks and dangers of permanent exterior ports being opened in a firewall, thereby maintaining corporate LAN security policies and minimizing the need for IT intervention.
+
+  </p>
+</details>
+
+<details>
+  <summary><a href="https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-01#section-6">AES Encryption</a></summary>
+
+  <p>
+
+  Using 128/192/256-bit AES encryption trusted by governments and organizations around the world, SRT ensures that valuable content is protected end-to-end from contribution to distribution so that no unauthorized parties can listen.
+
+  </p>
+</details>
+
+<details>
+  <summary><a href="./docs/features/packet-filtering-and-fec.md">Forward Error Correction (FEC) and Packet Filter API</a></summary>
+
+  <p>
+
+  [SRT 1.4](https://github.com/Haivision/srt/releases/tag/v1.4.0) sees the introduction of the _packet filter API_. What this means is that on the sender side, before the network packet is sent, it’s run through a packet filter so it can be processed. Similarly, on the receiver side, once a packet is received from the network, it runs through a filter and is processed on the other side. This plugin API allows users to write their own plugin for the SRT protocol, thereby extending its capabilities even further, offering all kinds of different packet filtering. When using the new filter, users can manipulate the resulting data any way they want for custom encryption, packet inspection, or accessing data before it’s sent for example.
+
+  The first plugin created as an example of what can be achieved with the packet filter API is for Forward Error Correction (FEC) which, in certain use cases, albeit very rarely, where there are very long-distance links and an excellent quality connection, can offer slightly lower latency than Automatic Repeat reQuest (ARQ). This plugin allows users to compare ARQ and FEC and allow three different modes to be run:
+
+   - ARQ only – retransmit of lost packets,
+   - FEC only – provides the overhead in the bandwidth needed for recovery on the receiver side,
+   - FEC and ARQ – if FEC doesn’t manage to recover a packet, you can still retransmit.
```

Comment:
- ARQ only
   - FEC only
   - FEC and ARQ

For more information, see [SRT Packet Filtering & FEC](/docs/features/packet-filtering-and-fec.md)

Reply:
Okay then. Some minor edits:

   - ARQ only – retransmits lost packets,
   - FEC only – provides the overhead needed for FEC recovery on the receiver side,
   - FEC and ARQ – retransmits lost packets that FEC fails to recover a packet.

Reply:
@stevomatthews FEC and ARQ – retransmits lost packets that FEC fails to recover ?

Reply:
Oops . . . yes, I missed that. Thanks!